rpcs3/Emu/System.cpp

```diff
@@ -2736,6 +2737,9 @@ void Emulator::CleanUp()
 {
 	// Deinitialize object manager to prevent any hanging objects at program exit
 	g_fxo->clear();
+
+	// deinitialize gun
+	evdev_gun_handler::shutdown();
```

Comment:
Needs proper gating. This controller doesn't exist if HAVE_LIBEVDEV is false as is the case on all non-linux platforms.

Reply:
Same goes for all other files that reference this class including cellGem.cpp